tensorflow/python/eager/def_function.py

```diff
@@ -557,6 +557,13 @@ def _get_tracing_count(self):
     result += self._stateful_fn.tracing_count if self._stateful_fn else 0
     return result
 
+  def experimental_get_tracing_count(self):
+    """
+    Return the tracing count in the recent calls for debugging.
+    Feature request in issue: https://github.com/tensorflow/tensorflow/issues/37323
```

Comment:
I don't think there is any need to reference the github issue in the comment.

Reply:
fixed.